target/linux/lantiq/base-files/etc/board.d/01_leds

```diff
@@ -55,6 +55,9 @@ WBMR300)
 	ucidef_set_led_switch "lan3" "LAN3" "wbmr300:green:lan3" "switch0" "0x20"
 	ucidef_set_led_default "router" "router" "wbmr300:green:router" "1"
 	;;
+RS353)
+        ucidef_set_led_default "dsl" "dsl" "$(lantiq_get_dt_led_chosen dsl)" "0"
```

Comment:
OK if I understand correctly, I have to specify to add manually this configuration in my documentation. I will fix that.